src/tools/types.ts

```diff
@@ -3,6 +3,7 @@ import { OptionValues } from '../modules/ModuleTypes';
 
 export interface Settings {
   searchUrl: string;
+  searchNewTab?: string;
```

Comment:
The name implies, that it can only have two values:
- It opens in a new tab
- It open in the same tab

Why would you need a string here?

Reply:
The name parameter of window.open is a string so I set it as such. There are some other values besides the two I'm using so I didn't really think about it. Although, I agree with you that the name implies a boolean so I'll make the change.

Reply:
Okay, I understand your intention. So, a different question: Will there ever be a different value for this besides ``_self`` and ``_blank``?

Reply:
I'm not sure. In any case it can be a boolean at least for now.

Reply:
Okay... For the future, you should read the docs for such questions:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a?retiredLocale=de#attr-target

I believe that it wouldn't make any sense to add anything besides ``_self`` and ``_blank``, so I agree and would make it a boolean.